app/views/support_interface/application_forms/immigration_status/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :browser_title, title_with_error_prefix('Edit applicant residency status', @immigration_status_form.errors.any?) %>
```

Comment:
```suggestion
<% content_for :browser_title, title_with_error_prefix('Edit applicant immigration status', @immigration_status_form.errors.any?) %>
```

Reply:
Thanks @frankieroberto 

Should we also change the row text in support? Not sure of the context why it uses residency instead.

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/50492247/158579215-9f85a976-e0d0-486e-9116-d0bcbc144f45.png">

Reply:
@avinhurry yeah, I spotted that too. I've fixed in https://github.com/DFE-Digital/apply-for-teacher-training/pull/6694 but feel free to fix it in this PR too. 👍

Reply:
I've fixed! You will need to rebase 😓. I'll review your PR